components/email/src/Email.svelte

```diff
@@ -192,6 +213,16 @@
       }
     });
   }
+
+  // This is for Email component demo purpose, where we want to show expanded threads by default on load.
+  // Show expanded thread view onload if `expandThreadOnLoad` prop is true.
+  $: if (activeThread && expandThreadOnLoad && !activeThread.expanded) {
+    console.log("Inside if: ", expandThreadOnLoad);
+    activeThread.expanded = true;
+    // Set `show_expanded_email_view_onload` to false after load
+    expandThreadOnLoad = false;
+  }
```

Comment:
Instead of doing this, couldn't we set `activeThread.expaned = show_expanded_email_view_onload` when we first fetch the activeThread? Essentially we're saying the default state should be expanded.

Reply:
`activeThread` gets updated reactively based on `thread`, whereas here we only want to set `activeThread.expanded = true` once on load, of the prop passed is true.